client/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ file(GLOB_RECURSE source_client src/*.cpp src/*.h)
 find_package(SFML COMPONENTS system window graphics audio network REQUIRED)
 
 include_directories(${SFML_INCLUDE_DIR})
-add_executable(ares_client ${source_client})
+add_executable(ares_client ${source_client} src/Game.cpp src/Game.h src/Player.cpp src/Player.h src/PlayerCommands.cpp src/PlayerCommands.h)
```

Comment:
Put that list in a variable like in the server's `CMakeLists.txt` and remove the `file` command